### DISCUSSION TITLE
Bartender 5.0.52 App Discussion

### DISCUSSION THREAD

**Discussion by krehel** [Homebrew/discussions#5427] [Casks]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Moving a discussion that was [here](https://github.com/Homebrew/homebrew-cask/pull/174106#issuecomment-2150828642) regarding community awareness and future analysis of the Bartender application rather than commenting a closed PR.

Edit: this is an ongoing development and new information may become available as this is worked through by the various parties.

cc: @core-code (thanks for the fantastic write-up!)

### Relevant casks

Bartender

**Comment by core-code** [comment]
there is now a blog post with "clarifications" on macbartender.com and surteesstudios.com but just like their response in reddit this just strengthens the concerns instead of alleviating them

> Recently, there was a change in the signing certificate for the app, and unfortunately, this change wasn’t communicated properly to you, our loyal users.

the problem is not that the certificate change was not communicated, the problem is that the sale of the company/app was not communicated

> Please rest assured that Bartender is signed by a valid Apple ID developer and notarized by Apple, which verifies the app to ensure it is free from malware. 

ugh. thats just wrong, it's trivial to notarize malware and we've even seen plenty of malware and spyware on the Mac App Store. saying things are alright because the app is notarized is an argument usually only PuP vendors make.

knowing Ben as a competent developer and upright person i question whether this really comes from him.

> To verify that this message is indeed from me, you can also see this post on the Surtees Studios website at [https://www.surteesstudios.com](https://surteesstudios.com/a-new-chapter/).

ugh. it could also prove that the domain www.surteesstudios.com was sold alongside the app and the other domains like bartender.com, macbartender.com ...


in fact it is likely that surteesstudios.com was also sold, because the account "Ben Surtees" <bens@surteesstudios.com> has been inactive since this situation has been going on. if he had not sold surteesstudios.com why would his e-mail account have vanished?

**Comment by alkusin** [comment]
Hello there.

Also, as I said in the Reddit thread, surteesstudios.com redirects to macbartender.com. Surtees Studios domain was updated 17h April, which means during or after acquisition, depending if we listen the new owner on Reddit that claimed they bought Bartender two months ago, or "Ben" (lol) who says in his ridiculous statement three months ago. And... Ben's letter sounds exactly like yesterday new owner replies on Reddit. To me, it seems like Applause made another shady move here. And, if you're wondering, no, I don't believe Earth is flat.

**Comment by neontuna** [comment]
This just gets weirder in my opinion.  I'm not much of a conspiracy theorist but you can't tell me that statement and this old blog post were written by the same person https://www.macbartender.com/b5blog/Bartender5-over-a-decade/

Why even mention that the post can be "verified" and then just link to another domain as proof.

**Reply by LanikSJ** [reply]
They likely don't have access to the original domain.

**Reply by danyalaytekin** [reply]
They're both still hosted by the same account according to certificate transparency logs. ~~I think~~ It's possible everything might actually be fine, apart from the more zealous analytics. In any case, I should probably have stopped using this app a while ago and this is a good way to be pushed into it.

**Reply by LanikSJ** [reply]
I made the same decision earlier today, due to my work I can't really take chances with apps and their questionable origin.

I've been using [Ice](https://github.com/jordanbaird/Ice) all day and it does pretty much everything Bartender did, aside from the increased spacing between the chevron and mission control buttons.  To me that's minor enough that I don't care.  It does everything I need it to do and that's good enough for me.

**Reply by misiektoja** [reply]
You mean menu bar spacing ? You can change it from CLI: https://www.reddit.com/r/MacOS/comments/16lpfg5/hidden_preference_to_alter_the_menubar_spacing/

I replaced Bartender with BetterTouchTool solution: https://community.folivora.ai/t/bartender-controversy-tutorial-on-how-to-manage-menubar-status-items-via-btt/37429

**Comment by misiektoja** [comment]
I personally think new owners prepared a statement, Ben made some small amendments, gave green light and they released it using his name (as it appears old domain has been sold as well). I doubt they would release sth without his approval as they could face legal actions.

**Reply by kytta** [reply]
> I doubt they would release sth without his approval as they could face legal actions.

This really only applies if the jurisdictions of Ben and the new owner are the same, which I highly doubt. Although the new owner is registered in the US, they seem like a shell company. Delaware, agent firm — good luck finding the real owner :(

**Reply by Ein-Tim** [reply]
@kytta from [Reddit](https://www.reddit.com/r/macapps/comments/1d7zjv8/comment/l7bckll/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button):
"Applause [who is listed as developer of Bartender on Setapp] is not independent. It's 100% owned by private equity firm, Vista Equity. Vista acquired Applause in 2017. More specifically, Vista Equity Partners Fund VI, llp (2016). [...]"

EDIT: That's not correct information. Sorry for spreading it.

**Reply by kytta** [reply]
> who is listed as developer of Bartender on Setapp

Well, the party who is listed as developer on Setapp is not necessarily the party that made the blog posts / statements seemingly authored by Ben. This does not prove nor disprove anything

**Reply by mukreb** [reply]
> @kytta from [Reddit](https://www.reddit.com/r/macapps/comments/1d7zjv8/comment/l7bckll/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button): "Applause [who is listed as developer of Bartender on Setapp] is not independent. It's 100% owned by private equity firm, Vista Equity. Vista acquired Applause in 2017. More specifically, Vista Equity Partners Fund VI, llp (2016). [...]"
this was a mistake, they accidentally confused applause.com with applause.dev, see follow up on [Reddit](https://new.reddit.com/r/macapps/comments/1d7zjv8/comment/l7dn57v/?context=3)

**Reply by Ein-Tim** [reply]
@mukreb Thanks for the heads up. I added a note to my original comment.
